unified-runtime/source/adapters/level_zero/usm.hpp

```diff
@@ -11,25 +11,83 @@
 
 #include "common.hpp"
 
+#include "ur_api.h"
 #include "ur_pool_manager.hpp"
+#include <set>
 #include <umf_helpers.hpp>
 
 usm::DisjointPoolAllConfigs InitializeDisjointPoolConfig();
 
+class EnqueuedPool {
```

Comment:
Is this class v1-specific? Can we move it to some common (with v2) file?

Reply:
No, we can move it, but at this point it'd be used only in v1.

Reply:
I've moved the `EnqueuedPool` implementation into `enqueued_pool.[cpp|hpp]` files.